tasks.go

```diff
@@ -8,9 +8,9 @@ import (
 
 const (
 	TaskRunning = "running"
-)
 
-var DefaultWaitInterval = 1 * time.Second
+	DefaultWaitInterval = 1 * time.Second
```

Comment:
this is actually on purpose as an exported var so you can set it in some init in your own code when implementing the client. it's a bit of a cheat to have a "const" in the package code but let a consumer of the package override it without having to add additional params to functions set/get funcs